client/state/current-user/reducer.js

```diff
@@ -25,6 +25,16 @@ import emailVerification from './email-verification/reducer';
 /**
  * Tracks the current user ID.
  *
+ * In development, if you are receiving Redux errors like this:
+ *
+ * 		Error: Given action "CURRENT_USER_ID_SET", reducer "id" returned undefined.
+ *
+ * Try:
+ *  - `Tail`ing php error logs for syntax errors
```

Comment:
I'd suggest to shorten this to something like `This is likely caused by a server-side error or stored state corruption/auth token expiry`